### PR TITLE
Gov shut down banner

### PR DESCRIPTION
**Pull request by rana22**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -54,6 +57,9 @@
     </style>
   </head>
   <body >
+    <!-- <div id="govAlertMsg">
```
Comment (iksheth) [inline public/index.html:60]: @rana22 can we these comments?

File: src/components/Header/NavHeaderWithAlert.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { withStyles } from '@material-ui/styles';
+import axios from 'axios';
+import HeaderView from './HeaderView';
+import NavBarContainer from '../NavBar/NavBarContainer';
+
+const urlLink = 'https://cbiit.github.io/nci-softwaresolutions-elements/banners/government-shutdown.html';
```
Comment (iksheth) [inline src/components/Header/NavHeaderWithAlert.js:11]: @rana22 this is url for analysis teams. Can we replace with one with **"https://cbiit.github.io/crdc-alert-elements/banners/government-shutdown.html"**

File: src/components/Layout/LayoutView.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import { withStyles, CssBaseline } from '@material-ui/core';
 import { HashRouter, Route, Switch } from 'react-router-dom';
-import { LinkBar } from 'bento-components';
 import aboutPageRoutes from '../../bento/aboutPagesRoutes';
 import resourceDropdownRoutes from '../../bento/resourceDropdownRoutes';
 import dataDropdownRoutes from '../../bento/dataDropdownRoutes';
-import Header from '../Header/HeaderView';
-import NavBar from '../NavBar/NavBarContainer';
+// import Header from '../Header/HeaderView';
```
Comment (iksheth) [inline src/components/Layout/LayoutView.js:7]: unused code. Can we remove it?

File: public/index.html [hunk 4]
```diff
@@ -32,6 +32,9 @@
     <script src="./injectEnv.js"></script>
     <script src="./js/session.js"></script>
 
+    <!-- Import web component -->
+    <script src="https://cbiit.github.io/nci-softwaresolutions-elements/components/include-html.js"></script>
```
Comment (iksheth) [inline public/index.html:36]: @rana22 looks like url is wrong. Correct url: "https://cbiit.github.io/crdc-alert-elements/components/include-html.js"

**Review by iksheth — CHANGES_REQUESTED** [review]

**Comment by rana22** [comment]
closing this PR

**Pull request closed without merging**